themes/custom/az_barrio/css/az-media-display.css

```diff
@@ -27,3 +27,7 @@
   width: 100%;
   max-width:100%;
 }
+
```

Comment:
Because of this https://git.drupalcode.org/project/bootstrap_barrio/-/blob/8.x-4.30/css/components/field.css#L6

Reply:
I think the issue exists on all entity reference fields. So should probably be moved to core.

Reply:
This ticket will block this PR so we can address the margin issue globally. 
https://github.com/az-digital/az_quickstart/issues/1995